RoddyCore/src/de/dkfz/roddy/client/cliclient/CommandLineCall.groovy

```diff
@@ -121,7 +115,19 @@ public class CommandLineCall {
      * [ a:a string, b:another string, c:123, ... ]
      * @return
      */
-    public List<String> getSetConfigurationValues() {
-        return optionsMap.get(RoddyStartupOptions.cvalues, []);
+    public List<ConfigurationValue> getSetConfigurationValues() {
+        def externalConfigurationValues = optionsMap.get(RoddyStartupOptions.cvalues, []);
+        def configurationValues = []
+        for (eVal in externalConfigurationValues) {
+            String[] splitIDValue = eVal.split(StringConstants.SPLIT_COLON);
+            //TODO Put in a better error checking when converting the split string to a configuration value.
+            //Remark, if a value contains : / colons, it will not work!
```

Comment:
We would have to allow for quoted colons: 

--cvalues="s3\\://a.b.c.d/bucket/object"